firebase-common/src/test/java/com/google/firebase/heartbeatinfo/DefaultHeartBeatControllerTest.java

```diff
@@ -119,12 +188,17 @@ public void generateHeartBeat_twoHeartBeatsSameUserAgent()
     heartBeatController
         .getHeartBeatsHeader()
         .addOnCompleteListener(executor, getOnCompleteListener);
-    String expected =
-        Base64.getEncoder()
-            .encodeToString(
-                "{\"heartbeats\":[{\"date\":[\"2015-03-02\",\"2015-03-01\"],\"agent\":\"test-agent\"}],\"version\":\"2\"}"
-                    .getBytes());
-    assertThat(getOnCompleteListener.await()).isEqualTo(expected);
+    JSONObject output = new JSONObject();
+    JSONArray array = new JSONArray();
+    JSONObject obj = new JSONObject();
+    ;
+    obj.put("agent", "test-agent");
+    obj.put("date", dateList);
+    array.put(obj);
+    output.put("heartbeats", array);
+    output.put("version", "2");
```

Comment:
Why are we at version "2"? isn't it the initial version of the payload? let's use "1" as per the payload doc

Reply:
ohhh as per the doc its 2